corehq/apps/app_manager/tests/test_views.py

```diff
@@ -311,6 +311,18 @@ def test_form_linking_context(self, _):
                 'name': 'm0 module > m0 form 0',
                 'unique_id': 'm0_module.m0_form_0'
             },
+            {
```

Comment:
I don't know how to link to it since it wasn't a change but should line 292 comment `multiselect module (can't be linked to)` be removed?